src/videojs-http-streaming.js

```diff
@@ -190,6 +255,13 @@ class HlsHandler extends Component {
           }
         });
       }
+
+      // Set up a reference to the HlsHandler from player.vhs. This allows users to start
+      // migrating from player.tech_.hls... to player.vhs... for API access. Although this
+      // isn't the most appropriate form of reference for video.js (since all APIs should
+      // be provided through core video.js), it is a common pattern for plugins, and vhs
+      // will act accordingly.
+      _player.vhs = this;
```

Comment:
We need this to map to deprecated `player.dash` and `player.hls`.

Reply:
Did we have a `player.hls`? I though just `player.tech_.hls`

Reply:
Uses could access `player.hls.representations`

Reply:
I forgot that it's already available (albeit deprecated). Since it still exists they should be able to continue to use it without issue.